Cargo.toml

```diff
@@ -18,6 +18,5 @@ features = ["regular", "size_14"]
 
 [patch.crates-io]
 # Change this to the revision of the bootloader you want to test.
-bootloader = { git = "https://github.com/rust-osdev/bootloader", rev = "960671c", package = "bootloader", version = "=0.11.0-alpha.0" }
-bootloader_api = { git = "https://github.com/rust-osdev/bootloader", rev = "960671c", package = "bootloader_api", version = "=0.1.0-alpha.0" }
-
+bootloader = { git = "https://github.com/rust-osdev/bootloader", branch = "next", package = "bootloader", version = "=0.11.0-alpha" }
+bootloader_api = { git = "https://github.com/rust-osdev/bootloader", branch = "next", package = "bootloader_api", version = "=0.1.0-alpha.0" }
```

Comment:
we might want to lock the revision of the bootloader to prevent it from breaking this kernel if the `next` branch changes the api. one alternative would be to not use a patch section, specify the dependency as a git dependency and then check in the Cargo.lock file. this would allow easy updating.

Reply:
Ah, I thought that the patch section respects the lockfile too. But git dependencies seem like a good idea!

Reply:
Oh yeah, you might be right about that.